keda/values.yaml

```diff
@@ -459,6 +459,27 @@ http:
   # -- The minimum TLS version to use for all scalers that use raw HTTP clients (some scalers use SDKs to access target services. These have built-in HTTP clients, and this value does not necessarily apply to them)
   minTlsVersion: TLS12
 
+## This setting let you enable profiling for all of the components of keda and in specific port you choose
+## This can be useful when trying to investigate errors like memory leak or cpu or even look at goroutines to understand better
+## This setting is disabled by default
+profiling:
+  operator:
+    # -- Enable profiling for keda operator
+    enabled: false
+    # -- Expose profiling on specific port
+    port: 8082
+  metricsServer:
+    # -- Enable profiling for keda metrics server
+    enabled: false
+    # -- Expose profiling on specific port
+    port: 8083
+  webhooks:
+    # -- Enable profiling for keda admission webhook
+    enabled: false
+    # -- Expose profiling on specific port
+    port: 8084
```

Comment:
```suggestion
## This setting lets you enable profiling for all of the components of KEDA and in the specific port you choose
## This can be useful when trying to investigate errors like memory leaks or CPU or even look at goroutines to understand better
## This setting is disabled by default
profiling:
  operator:
    # -- Enable profiling for KEDA operator
    enabled: false
    # -- Expose profiling on a specific port
    port: 8082
  metricsServer:
    # -- Enable profiling for KEDA metrics server
    enabled: false
    # -- Expose profiling on a specific port
    port: 8083
  webhooks:
    # -- Enable profiling for KEDA admission webhook
    enabled: false
    # -- Expose profiling on a specific port
    port: 8084
```